about.html

```diff
@@ -92,7 +92,7 @@ <h3 class="heading-heavy abt-logo-card">
             </div>
         </section>
 
-        <section class="abt-section-3 abt-section abt-section abt-past-events-section">
+        <section class="abt-section-2 abt-section abt-section abt-past-events-section section">
             <div class="">
                 <h3>See the past UNESCO Summit</h3>
                 <hr class="underline">
```

Comment:
- [ ] The two side by side images on the desktop do not show on your design, but do on mobile. It is required to be arranged on a row on the desktop and on a column on the mobile. Kindly look into it and fix.

### Your Design
![your-missing-img-desktop](https://github.com/Paulinelali/Microverse-capstone-one/assets/26597432/040d6b5c-4ca7-466a-a381-742e97ccb130)

### Original Design.
![temp-missing-img-desktop](https://github.com/Paulinelali/Microverse-capstone-one/assets/26597432/72351e52-4a47-4c95-b945-65fdb68d2226)